meta-jsdelivr/recipes-jsdelivr/jsdelivr-container-manager/files/jsdelivr-container-loader.sh

```
@@ -0,0 +1,392 @@
+                log "Starting existing container $container_name"
+                docker start "$container_name" > /dev/tty3 2>&1
+                ((started_count++))
+                continue
+            fi
+        fi
+
+        # Create volumes if needed
+        create_volumes "$container_name"
+
+        # Build advanced arguments
+        local volume_args=$(build_volume_args "$container_name")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)